### PR TITLE
TokenAge for MP JWT 2.1 For Issue #21186

### PR DESCRIPTION
**Pull request by simysc**
For issue #21186

### CODE REVIEW

**Comment by simysc** [comment]
#build

**Comment by ayoho** [comment]
Hi @simysc! Would you be able to go back and change all the tab characters in the changed files back to spaces? Our convention is to use spaces instead of tabs, and introducing them here can cause some headaches down the line.

**Comment by simysc** [comment]
@ayoho changed all the tab characters back to spaces, thanks a lot for your review!

**Comment by simysc** [comment]
#build

File: dev/com.ibm.ws.security.jwt/src/com/ibm/ws/security/jwt/internal/ConsumerUtil.java [hunk 1]
```diff
@@ -56,801 +57,852 @@
...
+                clockSkew = 0;
+            }
+        }
+
+        long tokenAge = 0;
+        // Take tokenAge value from server.xml
+        tokenAge = config.getTokenAge();
+        /**
+         * If tokenAge from server.xml is zero, then take the value of
+         * clock_skew from mpConfigProps
```
Comment (teddyjtorres) [inline dev/com.ibm.ws.security.jwt/src/com/ibm/ws/security/jwt/internal/ConsumerUtil.java:480]: "clock_skew" might have been a copy/paste error. We can change this to "tokenAge" some time later.

**Review by teddyjtorres — APPROVED** [review]

**Comment by teddyjtorres** [comment]
#libby

**Pull request merged**